iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer.inl

```diff
@@ -24,23 +24,23 @@ namespace iox
 {
 namespace rp
 {
-template <typename T>
-inline RelativePointer<T>::RelativePointer(ptr_t ptr, id_t id) noexcept
-    : BaseRelativePointer(ptr, id)
-{
-}
-
-template <typename T>
-inline RelativePointer<T>::RelativePointer(offset_t offset, id_t id) noexcept
-    : BaseRelativePointer(offset, id)
-{
-}
-
-template <typename T>
-inline RelativePointer<T>::RelativePointer(ptr_t ptr) noexcept
-    : BaseRelativePointer(ptr)
-{
-}
+// template <typename T>
```

Comment:
commented to address the violation, 
'Rule A12-1-6 : Derived classes that do not need further explicit initialization and require
all the constructors from the base class shall use inheriting constructors.'